scripts/event-geo.js

```diff
@@ -1,17 +1,16 @@
 'use strict'
+
+const htmlToText = require('html-to-text')
 const yaml = require('js-yaml')
 const fs = require('fs')
 const path = require('path')
-let p = path.join(__dirname, '..', 'locale', 'en', 'get-involved', 'events.md')
-let htmlToText = require('html-to-text')
 
-function load () {
-  let buf = fs.readFileSync(p)
-  let lines = buf.toString().split('\n')
```

Comment:
We can use `os.EOL` here as an alternative fix, but I find the proposed solution easier to grok.